### ISSUE TITLE
Eureur translation into French

### ISSUE THREAD

**Issue by IssoufKon** [africanmathsinitiative/R-Instat#7041]
**Describe the bug**
During my daily use of R-Instat in French, I noticed that many dialogues have translation problems. Indeed, either the translation is not complete (all options in the dialog are not translated), or it is not correct. I have added screenshots of some examples.

**Expected behaviour**
Review if possible all the translations in French. 

**Screenshots**
![Erreur02](https://user-images.githubusercontent.com/88491307/144844939-80b7c676-fe0e-4d13-bba4-480d7b9d27f3.GIF)
![erreur05](https://user-images.githubusercontent.com/88491307/144844949-a28a433f-2d7b-485f-800c-76fc775286ef.GIF)
![erreur06](https://user-images.githubusercontent.com/88491307/144844951-fd98995b-0bdc-4616-a745-45ded5fcca58.GIF)
![Eureur01](https://user-images.githubusercontent.com/88491307/144844953-3b91590e-25b2-47c9-b8f2-63d423759daf.GIF)
![Eurreur03](https://user-images.githubusercontent.com/88491307/144844955-875b7230-4d7a-430b-839a-f5af10c479e9.GIF)
![Eurreur04](https://user-images.githubusercontent.com/88491307/144844956-c227c957-3e12-4ac0-ac99-65be758da4a8.GIF)

**Details (optional):**
 - OS: [e.g. Windows 7]
 - R-Instat Version [e.g. 0.4.22]

**Comment by dannyparsons** [comment]
@IssoufKon thanks for the information about these.

There is a mix of different issues here:

1. Some translations are incorrect or need improving, @IssoufKon It would be great if you could help improve these on the CrowdIn site https://crowdin.com/project/r-instat
2. Some items are not being translated by R-Instat. This may be because the database has not been updated to include these terms yet. Or it may be an issue with the translation system. @lloyddewit would you be able to check this for some of the dialogs shown? For example, I notice that Bar Chart Options has a translation of "Options de graphique à barres" in CrowdIn
3. Some of the controls on the dialogs are not the right size to include the full translation and cut off the text. I think @rachelkg is/was working on these?

**Comment by rachelkg** [comment]
@IssoufKon and @dannyparsons, thanks so much for the information and screen shots. I had been working on the box sizing to ensure the French is displayed correctly. I have taken a break as I wanted to do my next run through once the data base had been updated and some of the dialogues due to be updated were finished. I will make a note of the issues highlighted here and rectify them. 

Thanks again and please don’t contact me with other issues. 

Best,

rachel

**Comment by lloyddewit** [comment]
>Some items are not being translated by R-Instat. This may be because the database has not been updated to include these terms yet. Or it may be an issue with the translation system. @lloyddewit would you be able to check this for some of the dialogs shown? For example, I notice that Bar Chart Options has a translation of "Options de graphique à barres" in CrowdIn

yes, I'll investigate which of the above issues can be fixed in the database.

**Comment by rachelkg** [comment]
@IssoufKon This information is really helpful in improving R-Instat, thanks for the examples but please could you start keeping a systematic record of the problems you find. The screen captures you have made are ideal, you could either keep adding them to this issue or you could keep an excel sheet to share. 

You can use the crowd in (https://crowdin.com/project/r-instat) site to improve the quality of the French translations. If you need help in using the crowd in site @N-thony could direct you.  

Please feel free to contact me if you have any questions. 

Thanks, 

Rachel
